components/filter/filter.js

```diff
@@ -489,6 +550,7 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 
 			switch (type) {
 				case 'd2l-filter-dimension-set': {
+					info.clearType = dimension.searchType === 'manual' ? 'manual' : dimension.clearType;
```

Comment:
If they've chosen to handle search manually, they also have to handle clear manually, since we aren't able to clear values that have been searched away.